src/SFA.DAS.AssessorService.Web.UnitTests/Validators/Standard/OptOutStandardVersionViewModelValidatorTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.AspNetCore.Http;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.Standards;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Validators.Standard;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+using SFA.DAS.Testing.AutoFixture;
+using System.Security.Claims;
+
+namespace SFA.DAS.AssessorService.Web.UnitTests.Validators.Standard
+{
+    public class OptOutStandardVersionViewModelValidatorTests
+    {
+        [Test, MoqAutoData]
+        public void MultipleApprovedStandardVersions_Passes()
+        {
+            var viewModel = new OptOutStandardVersionViewModel() { Version = "1.1", StandardReference = "ST999" };
+            var standardVersionOne = new StandardVersion() { Version = "1.1" };
+            var standardVersionTwo = new StandardVersion() { Version = "1.2" };
+
+            var httpContextAccessor = GetMockHttpContextAccessor();
+            var apiClient = GetMockApiClient(standardVersionOne, standardVersionTwo);
+
+            var sut = new OptOutStandardVersionViewModelValidator(httpContextAccessor.Object, apiClient.Object);
+
+            var result = sut.Validate(viewModel);
+
+            Assert.That(result.IsValid, Is.True);
+        }
+
+        [Test, RecursiveMoqAutoData]
+        public void OnlyOneApprovedStandardVersion_Fails_WithErrorMessage(OptOutStandardVersionViewModel viewModel,
+                                                                          StandardVersion version)
+        {
+            var httpContextAccessor = GetMockHttpContextAccessor();
```

Comment:
This would be better setup explicitly as per the above test as otherwise this is just random data

Reply:
Done. Recently I've become more sceptical of using AutoMoq in unit tests. It's too easy (and tempting) to let it construct all your objects, which makes it easy to gloss over details and write bloated tests with little effort. Good tests shouldn't be dependent on having fully setup objects